txstore.go

```diff
@@ -24,25 +24,32 @@ type TxStore struct {
 
 	Param *chaincfg.Params
 
-	masterPrivKey *hd.ExtendedKey
-	masterPubKey  *hd.ExtendedKey
+	accountMasterKey *hd.ExtendedKey
 
 	listeners []func(TransactionCallback)
 
 	Datastore
 }
 
 func NewTxStore(p *chaincfg.Params, db Datastore, masterPrivKey *hd.ExtendedKey) (*TxStore, error) {
-	mPubKey, err := masterPrivKey.Neuter()
+	// Derive the "account" key using Bip44
+	fourtyFour, err := masterPrivKey.Child(hd.HardenedKeyStart + 44)
```

Comment:
The the errors returned here are due to invalid keys this will always fail. It's a small chance, but it seems like we should handle some sort of logic for dealing with that. Perhaps we just require a new seed/masterPrivKey?